app/site.rb

```diff
@@ -81,6 +99,7 @@ def resource(path)
 		end
 
 		def git
+			ENV["SITE_PUBLIC_KEY_PATH"] = ssh_private_key_path.to_s
```

Comment:
Should this be <code>ENV["SITE_PUBLIC_KEY_PATH"] = ssh_*public*_key_path.to_s</code>?